scripts/rest_api.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
+SCRIPT=$(basename "${0}")
+ORGPATH=$(readlink -fn "${0}")
+ORGDIR=$(dirname "${ORGPATH}")
+#shellcheck source=scripts/helper_functions.sh
+source "${ORGDIR}/helper_functions.sh"
```

Comment:
The symbolic link /usr/local/bin/rest-cli refers to helper_functions.sh in the same location as the original rest_api.sh.